tests/check_amb_kf.c

```diff
@@ -67,12 +68,73 @@ START_TEST(test_lsq)
 }
 END_TEST
 
+START_TEST(test_sos_innov)
```

Comment:
In the case of this function, at least, it's just wrapping a bit of matrix algebra together. I think that would be overkill.